bundle/deno.jsonc

```diff
@@ -99,6 +99,9 @@
   },
   "lint": {
     "rules": {
+      "tags": [
+        "recommended"
+      ],
```

Comment:
I don't think it's necessary to explicitely mention the `recommended` set in every conf, it's already the default